packages/ag-charts-community/src/scene/group.ts

```diff
@@ -204,9 +201,7 @@ export class Group extends Node {
                 // clipBBox is in the canvas coordinate space, when we hit a layer we apply the new clipping at which point there are no transforms in play
                 const { width, height, x, y } = clipBBox;
 
-                if (consoleLog) {
-                    Logger.debug({ name, clipBBox, ctxTransform: ctx.getTransform(), renderCtx, group: this });
-                }
+                debug({ name, clipBBox, ctxTransform: ctx.getTransform(), renderCtx, group: this });
```

Comment:
We may want to put some guard around this code to avoid unecessary garbage generation just by construction of the parameter object?